static/js/tools/timeline_page.test.tsx

```diff
@@ -0,0 +1,118 @@
+jest.mock("axios");
+jest.mock("../chart/draw");
+
+import Enzyme, { mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { when } from "jest-when";
+
+import React from "react";
+import axios from "axios";
+import * as d3 from "d3";
+
+import { Page } from "./timeline_page";
+import { drawGroupLineChart } from "../chart/draw";
+import { DataGroup, PlotParams } from "../chart/base";
+import hierarchy from "../../data/hierarchy_top.json";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const globalAny: any = global;
+
+test("Single place and single stats var", () => {
+  globalAny.window = Object.create(window);
+  document.body.innerHTML = '<button id="download-link"></div>';
```

Comment:
update closing tag to button also (was this copied from the template? that should be updated too).